src/components/NumericSensorReadout.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Typography, ExpansionPanelDetails } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { hot } from 'react-hot-loader';
+import PropTypes from 'prop-types';
+
+const styles = (theme) => ({
+  expansionPanelSubDetails: {
+    color: theme.palette.text.secondary,
+    paddingTop: theme.spacing(0),
+    paddingBottom: theme.spacing(1),
+  },
+});
+
+const NumericSensorReadout = ({
+  title,
+  readings,
+  unit,
+  classes,
+}) => (
+  <>
+    <ExpansionPanelDetails>
```

Comment:
I don't love that we assume that this component is used inside an ExpansionPanel. However, if you don't put things inside their own `ExpansionPanelDetails`, it tries to smash them together side-by-side for some reason. I thought this solution would work ok for now.